src/api/client.ts

```diff
@@ -97,61 +93,16 @@ async function requestIdTokenHandler(config: InternalAxiosRequestConfig) {
   return config;
 }
 
-function responseIdTokenHandler(error: AxiosError) {
-  if (error?.config) {
-    error.config.forceRefreshIdToken =
-      error.config.authWithIdToken && error.response?.status === 401;
-  }
-  throw error;
-}
-
-function isInternalUpstreamServerError(
-  e: any,
-): e is InternalUpstreamServerError {
-  return 'errorCode' in e && 'shortNorwegian' in e;
-}
+function responseErrorHandler(
+  error: AxiosError,
+): Promise<ErrorResponse | HttpErrorResponse> {
+  const errorResponse = parseErrorResponse(error);
 
-function responseErrorHandler(error: AxiosError) {
-  if (shouldSkipLogging(error)) {
-    return Promise.reject(error);
+  if (!shouldSkipLogging(error)) {
+    notifyError(error);
```

Comment:
With this change does `notifyError` get called more often than before? Previously, the `'timeout'` and `'networkError'` case didn't call it.

Reply:
Good question. Should we notify if there's a timeout or network error, for requests that shouldn't be skipped?